src/Valtuutus.Data.SqlServer/SqlServerDataReaderProvider.cs

```diff
@@ -63,7 +74,7 @@ public async Task<List<AttributeTuple>> GetAttributes(EntityAttributeFilter filt
 #endif
             var queryTemplate = new SqlBuilder()
                 .FilterAttributes(filter)
-                .AddTemplate(SelectAttributes);
+                .AddTemplate(string.Format(SelectAttributes, _dbOptions.Schema, _dbOptions.AttributesTableName));
```

Comment:
I can probably cache this and other queries so I don't need to generate a new string everytime